cesium/atomic
# refactor: sidebar

## Description
The purpose of this PR is to refactor the sidebar, since it was using alpine and other outdated stuff.

## Related Issues
N/A

## Steps to reproduce or test
- mix phx.server
- Open at localhost:4000
- And see the sidebar in desktop and mobile views
